src/python/grpcio_tests/tests/reflection/_reflection_servicer_test.py

```diff
@@ -50,6 +50,16 @@ def _file_descriptor_to_proto(descriptor):
 
 class ReflectionServicerTest(unittest.TestCase):
 
+    # TODO(https://github.com/grpc/grpc/issues/17844)
+    # Bazel + Python 3 will result in creating two different instance of
+    # DESCRIPTOR for each message. So, the equal comparison between protobuf
+    # returned by stub and manually crafted protobuf will always fail.
+    def _assert_sequence_of_proto_equal(self, x, y):
+        self.assertSequenceEqual(
+            list(map(lambda x: x.SerializeToString(), x)),
```

Comment:
I'd suggest `tuple(proto.SerializeToString() for proto in x)`

Reply:
That's a good idea. TIL.